documentation/ide-vscode-setup.html

```diff
@@ -0,0 +1,189 @@
+            for more information.
+        </p>
+        <p>
+            Visual Studio Code or VS Code is a fast editor and ships with great editing features. It includes support for debugging,
+            embedded Git control, syntax highlighting, intelligent code completion, snippets, and code refactoring. 
+            In part thanks to the huge ecosystem of extensions that enhance and enrich the experience in VS Code, 
+            turning it into a super smart text editor, fast and lightweight. This guide focuses on the bare minimum that 
+            is needed to get started Openfire source code with VS Code.
+        </p>
+
+        <h2>Prepare your environment</h2>
+        <p>
+            The rest of this guide will assume you have at least those parts installed:
+            <ul>
+                <li><b>JAVA Development Kit (version 8)</b> </li>
+                <ul>
+                    Java 8 (JDK 1.8) must be installed and setup on your machine. To test the installation,
+                    open a shell in a Unix or a MS-DOS prompt in Windows. Check your version of
+                    Java with "java -version" -- it must version 1.8 or greater.
+                    If Java isn't installed, download a copy from the
+                    <a href="http://www.oracle.com/technetwork/java/javase/overview/index.html">Java website</a>.
+                    <p>
+                        <font color="red">Important!</font> -- the Openfire build tool needs to know
+                        where Java is installed on your system. You must configure the "JAVA_HOME"
+                        environment variable to point to the correct directory. 
+                    <p>
+                 </ul>
+                <li><b>Maven</b></li> 
+                    <ul>
+                        <p>
+                            The Openfire build process uses Maven, so that tool must be installed
+                            and configured on your computer. First download Maven from: 
+                            <a href="https://maven.apache.org/">https://maven.apache.org</a>. Next, follow
+                            the <a href="https://maven.apache.org/users/index.html">usage instructions</a>.
+                        <p>
+                    </ul>
+                <li><b>Copy of the  Openfire source code available locally </b></li>
+                    <ul>
+                        <p>
+                            You can get the Openfire source code by downloading (clonning) a 
+                            source code on the Ignite Realtime GitHub Repository.
+                             <a href="https://github.com/igniterealtime/Openfire">Source Code</a>. 
+                        </p>
+                    </ul>
+                <li><b>Visual Studio Code</b></li> 
+                    <ul>
+                        <p>
+                            Downloading both the versions of Visual Studio Code is pretty simple. 
```

Comment:
I would just leave "Downloading Visual Studio Code is pretty simple." or i would just remove that sentence as it doesn't give useful information. Anything is easy to download nowadays :)